packages/commons/src/email-manager/emailManager.ts

```diff
@@ -76,7 +82,31 @@ export function initSesMailManager(awsConfig: AWSSesConfig): EmailManagerSES {
         },
       };
 
-      await client.send(new SendEmailCommand(input));
+      try {
+        await client.send(new SendEmailCommand(input));
+      } catch (err) {
+        if (!skipError) {
+          throw err;
+        }
+
+        /*
+          Temporary Hotfix: https://pagopa.atlassian.net/browse/PIN-6514 
+          We want to avoid treating the TooManyRequestsException as a fatal error 
+          when the rate limit is reached with the current configuration.
+          The following statement skips the TooManyRequestsException error thrown by the AWS SES client.
+
+          For more details about the errors and best practices to handle them, refer to:
+          - AWS SES client: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-client-ses/Class/SES/
+          - AWS SDK Error Handling: https://aws.amazon.com/blogs/developer/service-error-handling-modular-aws-sdk-js/  
+        */
+        if (err instanceof TooManyRequestsException) {
+          logger.warn(
+            `AWS SES error with name ${err.name} was thrown, it will not be considered fatal, email notification not sent; Error details: ${err.message}`
```

Comment:
Not specific for notifications anymore
```suggestion
            `AWS SES error with name ${err.name} was thrown, skipTooManyRequestsError is true so it will not be considered fatal, but the email is NOT sent; Error details: ${err.message}`
```